docs/docs.md

```diff
@@ -834,4 +834,12 @@ def dynamics_client(request):
 #### dynamics_cache
 
 An instance of the cache used by the client. Can be either an instance of the SQLiteCache
-that comes with the library, or Django's cache if it's installed.
\ No newline at end of file
+that comes with the library, or Django's cache if it's installed.
+
+#### environ
```

Comment:
`environ` fixture is in `test/conftest.py`, so it won't be included with the library when installed from pip, so this does not need to be documented.

Reply:
Got it - removed.